travel-ban/terrorism-incidents/wikipedia-scrape.py

```diff
@@ -0,0 +1,106 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import Select
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+import os
+import shutil
+import json
+
+def start_driver():
+	return webdriver.Firefox()
+
+def get_page_links(driver, rootUrl='https://en.wikipedia.org/wiki/List_of_terrorist_incidents', all_years=map(str, range(1970, 2018))):
```

Comment:
Neither root_url or all_years should have default values, those should be flexible and specified by the user and break when they are not.